app/Http/Controllers/UserManageApiController.php

```diff
@@ -115,15 +115,33 @@ public function postSignup()
             $user->lname = $lname;
             $user->token = $token;
             $user->save();
+            $user_id = $user->id;
+
+            try {
+
+                //Send email to user for email account verification
+                Mail::queue('email.signup', array('token' => $token), function ($message) use ($email, $fname) {
+                    $message->subject('Welcome to the Madison Community');
+                    $message->from('sayhello@opengovfoundation.org', 'Madison');
+                    $message->to($email); // Recipient address
+                });
+
+                return Response::json(array( 'status' => 'ok', 'errors' => array(), 'message' => 'An email has been sent to your email address.  Please follow the instructions in the email to confirm your email address before logging in.'));
+
+            } catch (\Exception $e) {
+
+                //auto verify the user and log them in
+                //in case sending the email fails
+                $user = User::where('id', $user_id)->first();
```

Comment:
@dkd903 You shouldn't need to re-find the user here, you've still got a reference to `$user` from above which is valid in this scope.